src/azure-cli/azure/cli/command_modules/alertsmanagement/_params.py

```diff
@@ -0,0 +1,182 @@
+        c.argument('target_resource_group', options_list=['--target-resource-group'],
+                   required=False, help='Filter by resource group')
+        c.argument('monitor_service', monitor_service_arg_type, required=False)
+        c.argument('monitor_condition', monitor_condition_arg_type, required=False)
+        c.argument('severity', severity_arg_type, required=False)
+        c.argument('state', state_arg_type, required=False)
+        c.argument('alert_rule', alert_rule_arg_type, required=False)
+        c.argument('time_range', time_range_arg_type, required=False)
+        c.argument('custom_time_range', custom_time_range_arg_type, required=False)
+
+    with self.argument_context('alertsmanagement alert show-history') as c:
+        c.argument('alert_id', options_list=['--alert-id'], required=True, help='Id of the alert to fetch it’s history')
+
+    with self.argument_context('alertsmanagement smart-group list') as c:
+        c.argument('sort_by', sortby_arg_type, required=False)
+        c.argument('sort_order', sort_order_arg_type, required=False)
+        c.argument('time_range', time_range_arg_type, required=False)
+
+    with self.argument_context('alertsmanagement  smart-group show') as c:
+        c.argument('smart_group_id', smart_group_id_arg_type, required=True)
+
+    with self.argument_context('alertsmanagement  smart-group show-history') as c:
+        c.argument('smart_group_id', smart_group_id_arg_type, required=True)
+
+    with self.argument_context('alertsmanagement action-rule list') as c:
+        c.argument('resource_group_name', resource_group_name_arg_type, required=False)
+        c.argument('target_resource', target_resource_arg_type, required=False)
+        c.argument('target_resource_type', target_resource_type_arg_type, required=False)
+        c.argument('target_resource_group', target_resource_group_arg_type, required=False)
+        c.argument('severity', severity_arg_type, required=False)
+        c.argument('monitor_service', monitor_service_arg_type, required=False)
+        c.argument('impacted_scope', options_list=['--impacted-scope'], required=False,
+                   help='Gets all actions rules in a subscription filter by impacted scope')
+        c.argument('description', options_list=['--description'], required=False,
+                   help='Gets all actions rules in a subscription filter by description')
+        c.argument('alert_rule', alert_rule_arg_type, required=False)
+        c.argument('action_group', options_list=['--action-group'], required=False,
+                   help='Gets all actions rules in a subscription filter by action group')
+        c.argument('name', options_list=['--name'], required=False,
+                   help='Gets all actions rules in a subscription filter by action rule name')
+
+    with self.argument_context('alertsmanagement action-rule show') as c:
+        c.argument('resource_group_name', resource_group_name_arg_type, required=True)
```

Comment:
The argument should be 'resource_group'. It should look as follows in help:

    --resource-group -g         [Required] : Name of resource group. You can configure the default
                                             group using `az configure --defaults group=<name>`.

Reply:
Required changes have been done. Thanks!